### PR TITLE
Cleanup lint and files

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
Moved all Workflow files to the workflow folder, and any reference of them fixed. 
Removed all lint warnings by reformating the code to be properly within lint standards
Removed Truthy Lint warings by disabling them, Was complaining about True vs true and False vs false.

### CODE REVIEW

File: .github/workflows/.yamllint.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ rules:
   indentation:
     level: warning
     indent-sequences: consistent
+  truthy: disable
```
Comment (kedark3) [inline .github/workflows/.yamllint.yml:11]: I think we should standardize this to something. Internally we have "True/False" - with first letter always in caps. Can we mass replace those in code base. I do not want to disable this option if possible.

**Review by kedark3 — CHANGES_REQUESTED** [review]
Requesting one change, if we can do it. Everything else looks good.

File: .github/workflows/.yamllint.yml (hunk 1 above)
Comment (sean-m-sullivan) [reply .github/workflows/.yamllint.yml:11]: I just read up on this based on your comment, I will look into adding allowed values of "True/False" in addition to "true/false" as per https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy

Comment (sean-m-sullivan) [reply .github/workflows/.yamllint.yml]: Updated to allow True/False/true/false changed a yes/no from tower_role/tests/test.yml to pass the test

**Review by Tompage1994 — APPROVED** [review]

**Review by kedark3 — APPROVED** [review]

**Pull request merged**